sdk/communication/CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+
+After setting up the environment variables and setting `AZURE_TEST_MODE` to `LIVE`, you can run the tests in the Test Explorer.
+
+### Record mode
+
+RECORD mode is similar to LIVE mode because it also hits an actual resource. In addition to hitting the resource, RECORD mode will also record the successful calls to the service in json format. As mentioned, the recordings are stored in the test package under `SessionRecords\<TestFileName>`.
+
+If you would like to generate new recordings for a test, setting `AZURE_TEST_MODE` to `RECORD`, reopen Visual Studio, and run the tests in the Test Explorer.
+
+### Managed Identity Tests
+
+If you ran the tests in RECORD mode, you may have noticed that the files inside the `SessionRecords` folder were updated. If any of the tests failed, you will see the error message in your terminal logs.
+
+The most probable thing is that the managed identity tests will fail at first. This is because we haven't set up any managed identity credentials for the DefaultAzureCredential object inside the tests to reference to. There are multiple ways of creating a managed identity credential.
+
+One of the easiest ways is to install the [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli) and run the `az login` command. If you are listed as a contributor of the resource you are testing against, this should be enough for the DefaultAzureCredential object to get the corresponding Azure Active Directory credentials you need.
+
+Another way to authenticate is to set up 3 environment variables called `AZURE_CLIENT_ID`, `AZURE_TENANT_ID` and `AZURE_CLIENT_SECRET` and set their values to the ones from a registered Azure Active Directory application that is linked to the resource you are testing against.
+
+If you are testing against a personal resource, you can check the [Managed Identity Quickstart Guide for ACS](https://docs.microsoft.com/azure/communication-services/quickstarts/managed-identity-from-cli) for an easy ramp-up process.
+
+For a more in-depth look on how to authenticate using managed identity, refer to the [Azure Identity client library for .Net](https://docs.microsoft.com/en-us/dotnet/api/overview/azure/identity-readme?view=azure-dotnet) documentation. This document also has more ways for you to authenticate using the DefaultAzureCredential object besides the ones we discussed in this contributing file.
+
+## Submitting a Pull Request
+
+The easiest way for you to test without worrying about breaking changes you may cause is to create a fork from the [.Net Azure SDK repo](https://github.com/Azure/azure-sdk-for-net). After downloading your repo, make sure to add the original repo as an upstream. To do this, use the `git remote add upstream` command followed by the repo's URL. 
```

Comment:
I think we can actually finish this command for them right?
`git remote add upstream https://github.com/Azure/azure-sdk-for-net.git`

Reply:
Great suggestion :)